### DISCUSSION TITLE
Is there a matching relation between Kong 2.5.X and Kubernetes version?

### DISCUSSION THREAD

**Discussion by tipdancer** [Kong/kong#9110] [Help]
For Kong Gateway (OSS), it has the following requirements:
Prerequisites
A Kubernetes cluster, v1.19 or later
kubectl v1.19 or later
Section "INstall on Kubernetes" at https://docs.konghq.com/gateway/2.6.x/install-and-run/kubernetes/

Here is my question: does Kong 2.5.X have such limitation like only supporting certain version of Kubernetes?
If Kong 2.5.X can support all version of Kubernetes, why Kong 2.6.X and the later version require Kubernets v1.19 and newer?

**Comment by hanshuebner** [comment]
Hello @tipdancer,

thank you for your question.  With Kong version 2.6, the documentation was restructured and improved to specify the minimum supported versions of Kubernetes.  "Supported" does not automatically mean that Kong would not run at all with older versions, but as we can do only limited testing, we're now restricting our support to the versions specified in the documentation.  It is not impossible that even Kong 2.5.x would not work with all versions of Kubernetes - After all, the documentation only specifies that Kong is compatible with all _distributions_, not _versions_ of Kubernetes.

Kind regards,
Hans
